tests/foreman/cli/test_repository.py

```diff
@@ -908,6 +908,40 @@ def test_positive_synchronize_docker_repo(self, repo, module_product, module_org
             ]
         )
 
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
```

Comment:
Mentioned BZ also has one upgrade related case attached, so it'd be better to cover this for upgrades
```suggestion
    @pytest.mark.tier2
    @pytest.mark.upgrade
    @pytest.mark.parametrize(
```